CHANGELOG.md

```diff
@@ -1,4 +1,48 @@
-## 3.83.0 (Unreleased)
+## 3.84.0 (Unreleased)
+
+FEATURES:
+* **New Data Source:** `google_secret_manager_secret` ([#9983](https://github.com/hashicorp/terraform-provider-google/pull/9983))
+
+IMPROVEMENTS:
```

Comment:
Any idea why these didn't make it?

* https://github.com/hashicorp/terraform-provider-google/pull/10033
* https://github.com/hashicorp/terraform-provider-google/pull/10038
* https://github.com/hashicorp/terraform-provider-google/pull/10047

Reply:
Oh no, they shouldn't be in beta. I thought I cut the release from the Tuesday night run of the tests and cherry-picked the disk interface change into both branches

Reply:
Oh, I think I know what happened.

When I generated TPG I ran the changelog gen script *before* cherry-picking the disk interface fix and added it's changelog manually.

When I did TPGB I cherry-picked first thinking it would be smart, but it made the `merge-base` find the wrong commit and generate changelogs for a bunch of extra PRs that aren't in this release.

Reply:
Got it! I think this is also what happens if the build number has a space prepended, which at least for me happens every time I copy it.

Reply:
Should be fixed